azure-pipelines.yml

```diff
@@ -26,10 +26,10 @@ jobs:
   # ******************************************************************************
   # Installs test suite
   # ******************************************************************************
-  - template: azure-pipelines-test-job.yml
-    parameters:
-      name: Installs
-      testScript: tasks/e2e-installs.sh
+  # - template: azure-pipelines-test-job.yml
```

Comment:
Why are these commented out?

Reply:
the tests are and have been failing for some time - we don't have a policy set up on the repo protecting from merging things that break the tests, it's a problem that have been growing for some time - so I've disabled them for now to see if any of the working tests were broken.

Reply:
@iansu closing this comment as we have focus on it already